.github/workflows/CI.yml

```diff
@@ -0,0 +1,12 @@
+name: CI
+on: push
+
+jobs:
+  molecule:
+    name: Run molecule test on Ansible role
+    runs-on: ubuntu-20.04 
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - uses: peopledoc/molecule-action@v2
```

Comment:
Hi, there are a lot of good stuff in your PR, but we can't merge this line (and some others specific for another repository).